### PR TITLE
Feature/aatf nil return

### PR DESCRIPTION
**Pull request by JB12298**

### CODE REVIEW

File: src/EA.Weee.Web/Areas/AATFReturn/Controllers/SelectReportOptionsNilController.cs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            this.requestCreator = requestCreator;
+        }
+
+        [HttpGet]
+        public virtual async Task<ActionResult> Index(Guid organisationId, Guid returnId)
+        {
+            using (var client = apiClient())
+            {
+                var returnData = await client.SendAsync(User.GetAccessToken(), new GetReturn(returnId));
+                var viewModel = mapper.Map(new ReturnDataToSelectReportOptionsNilViewModelMapTransfer()
```
Comment (SamGoodeySFW) [inline src/EA.Weee.Web/Areas/AATFReturn/Controllers/SelectReportOptionsNilController.cs]: Unit test to make sure mapper is called with correct values assigned to transfer

File: src/EA.Weee.Web/Areas/AATFReturn/ViewModels/SelectReportOptionsNilViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿namespace EA.Weee.Web.Areas.AatfReturn.ViewModels
+{
+    using System;
+    using System.Globalization;
+
+    public class SelectReportOptionsNilViewModel
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/AATFReturn/ViewModels/SelectReportOptionsNilViewModel.cs]: Possible to inherit this model from the ReturnViewModelBase is it has many of these properties available?

File: src/EA.Weee.Web/Areas/AATFReturn/Controllers/SelectReportOptionsNilController.cs (hunk 1 above)
Comment (JB12298) [reply src/EA.Weee.Web/Areas/AATFReturn/Controllers/SelectReportOptionsNilController.cs]: Added

File: src/EA.Weee.Web/Areas/AATFReturn/ViewModels/SelectReportOptionsNilViewModel.cs (hunk 2 above)
Comment (JB12298) [reply src/EA.Weee.Web/Areas/AATFReturn/ViewModels/SelectReportOptionsNilViewModel.cs]: Yep, I've made changes so it inherits

**Pull request merged**